src/ucp/rndv/rndv.c

```diff
@@ -1816,6 +1866,29 @@ static ucs_status_t ucp_rndv_send_start_put_pipeline(ucp_request_t *sreq,
     return UCS_OK;
 }
 
+static UCS_F_ALWAYS_INLINE ucp_request_t*
+ucp_rndv_rma_get_zcopy_req(ucp_request_t *req)
+{
+    ucp_request_t *rndv_req;
+
+    if (req->send.state.uct_comp.func == ucp_rndv_recv_frag_get_completion) {
```

Comment:
can we check FRAG flag instead?

Reply:
unfortunately, we don't set FRAG for GET Zcopy